bundles/binding/org.openhab.binding.caldav-personal/src/main/java/org/openhab/binding/caldav_personal/internal/CalDavBinding.java

```diff
@@ -220,11 +227,17 @@ private void updateItemsForEvent() {
         for (String item : bindingProvider.getItemNames()) {
             CalDavConfig config = bindingProvider.getConfig(item);
             List<CalDavEvent> events = eventCache.get(config.getUniqueEventListKey());
-            if (events == null && this.calDavLoader != null) {
+            if (events == null) {
+                logger.debug("No events found in event cache. Attempting to load from calendar.");
```

Comment:
Maybe we want to log the item, too. Like you did in [L238](https://github.com/openhab/openhab1-addons/pull/5391/files#diff-f1debbea460b13c062ee08716dfa2305R238). Wdyt?

`logger.debug("No events found in event cache for item '{}'. Attempting to load from calendar.", item);`

Reply:
Good idea. I'll add that.